### PR TITLE
Setup and mobile first

### PR DESCRIPTION
**Pull request by sagieramos**
### This project includes:
- Interactive social icons

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+I would like to thank Microverse and my coding partner for their invaluable support and contribution to the codebase.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE) licensed.
+
```
Comment (iambenkis) [inline README.md:123]: - [ ] Good job setting up you documentation file 👍 , however the `MIT` license you provided has a broken link , Kindly fix it to make it professional ✋ 
Capture

File: index.html [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+            <aside class="sidebar">
+                <menu class="drop-menu">
+                    <ul class="menu">
+                        <li class="menu-list"><a href="#">Program</a></li>  
+                        <li class="menu-list"><a href="#">About</a></li>  
+                        <li class="menu-list"><a href="#">Contact</a></li>         
+                    </ul>
+                </menu>
+            </aside>
+        </nav>
```
Comment (iambenkis) [inline index.html:29]: - [ ] Keep doing the great work 👍

File: index.html [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+                hesitate to contact me.
+            </p>
+            <h2>LET’S CONNECT</h2>
+            <ul>
+                <li><a href="#"><img src="./assets/twt.svg" alt="Twitter"></a></li>
+                <li><a href="#"><img src="./assets/Linkedin.svg" alt="LinkedIn"></a></li>
+                <li><a href="#"><img src="./assets/meduim.svg" alt="Medium"></a></li>
+                <li><a href="#"><img src="./assets/github.svg" alt="GitHub"></a></li>
+                <li><a href="#"><img src="./assets/angellist.svg" alt="AngelList"></a></li>
+              </ul>
```
Comment (iambenkis) [inline index.html:45]: - [ ] Keep doing the great work 👍

**Review by iambenkis — CHANGES_REQUESTED** [review]
# STATUS: REQUIRED CHANGES :recycle::recycle: 

Hi @sagieramos   
Good job so far! I know you worked hard to implement this project, I really appreciate the effort you put in! But there are some issues that you still need to work on to go to the next project but you are almost there! 

almost-there

## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- Good commit history :heavy_check_mark: 
- Follow GitHub flow :heavy_check_mark: 
- Stick to the Figma design :heavy_check_mark: 

## Required changes :recycle: :recycle: 

- [ ] Check the comments under the review.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear. 
Please ping me @iambenkis  when you comment, so I can receive the notification or use Slack with the same name._

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (sagieramos) [reply README.md:123]: Thanks for pointing this out. It has been resolved

File: index.html (hunk 2 above)
Comment (sagieramos) [reply index.html:29]: Thank you

File: index.html (hunk 3 above)
Comment (sagieramos) [reply index.html:45]: Thank you

**Review by franclobo — APPROVED** [review]
Hi @sagieramos, 

# STATUS: APPROVE 😃 

git-merge

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights

- No linter errors ✔️ 
- Descriptive PR ✔️ 
- Correct Github flow ✔️ 
- Best practices of HTML & CSS ✔️ 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**